src/lib/twemoji-utils.js

```diff
@@ -0,0 +1,19 @@
+export function toHTMLElement(text) {
+	const text_split = text.split("");
+	
+	for (const i in text_split) {
+		if (text_split[i] == "<") {
+			text_split[i] = "&lt";
+		} else if (text_split[i] == ">") {
+			text_split[i] = "&gt";
+		} else if (text_split[i] == "\n") {
+			text_split[i] = "<br>";
+		}
+	}
+	
+    const span = document.createElement("span");
+    const p = document.createElement("p");
+    p.innerText = text_split.join("");
+    span.appendChild(p);
+    return span;
+}
```

Comment:
probably a better way to do this:
```suggestion
export function toHTMLElement(text) {
    const filteredText = text
        .replaceAll("&", "&amp;")
        .replaceAll("<", "&lt;")
        .replaceAll(">", "&gt;")
        .replaceAll('"', "&quot;")
        .replaceAll("'", "&apos;");
    
    const span = document.createElement("span");
    const p = document.createElement("p");
    p.innerText = filteredText;
    span.appendChild(p);
    return span;
}
```
and add `white-space: pre-wrap;` as a style to `.post-content` (so newlines and duplicated spaces are preserved)